linux:
  hosts:
  children:
    rocky:
      hosts:
        10.0.5.78:
          hostname: Rocky1
          lan_ip: 10.0.5.10
        10.0.5.79:
          hostname: Rocky2
          lan_ip: 10.0.5.11
        10.0.5.80:
          hostname: Rocky3
          lan_ip: 10.0.5.12
      vars:
        device: ens192
        pub_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCy6Tm0R3VOp26FTJIFwXjM/gfCC93wyUF1aBhpwEq3tc3WWZu50uVAijb3WXMdOw2rHuyP7rMfwo/Eq0Hqwjus4Uv7wLweiDVcnJsP/IuJMECLKQGBOxL0uuVlCQIPgYFynEMUUldOhXShT3GHz3OGE9y7QmwuGZIZDgtmL32xczPEZc00H5ljt6SSU+026mbsj0nF9StaAKJeZtNKPfM1CjLwif7xaEEkL35FWp30BQI99KxnA74y4Kr6uJ1WD0DfJAz/HVaO/G43MoGf8PWORJC6EBDK6HIrdx1JIm5AIl4lh1I+aLhJYm2yXIzXbLAGDV8F/3QgdIgRJXG1co3q9dAEmPjBeHRGxxkC6GKYIcNtZHJpYSTVtVLuCp0TMtr/WyXTA015bKK3Iu7FZ66jiwWrBT6muJle7RvwPsaEkP0eZZortap8bZUhnHZYuDM2BjZTcYJULcMuaS436sXMXqUJ/uPnOD5rH3m+r7IfMTVAzuIU9d2LKiLqOVH8kL8="
        ansible_user: wayche
        prefix: 24
        gateway: 10.0.5.2
        name_server: 10.0.5.5
        domain: blue1.local
    ubuntu:
      hosts:
        10.0.5.29:
          hostname: ubuntu1
        10.0.5.30:
          hostname: ubuntu2
      vars:
        device: ens160
        pub_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCy6Tm0R3VOp26FTJIFwXjM/gfCC93wyUF1aBhpwEq3tc3WWZu50uVAijb3WXMdOw2rHuyP7rMfwo/Eq0Hqwjus4Uv7wLweiDVcnJsP/IuJMECLKQGBOxL0uuVlCQIPgYFynEMUUldOhXShT3GHz3OGE9y7QmwuGZIZDgtmL32xczPEZc00H5ljt6SSU+026mbsj0nF9StaAKJeZtNKPfM1CjLwif7xaEEkL35FWp30BQI99KxnA74y4Kr6uJ1WD0DfJAz/HVaO/G43MoGf8PWORJC6EBDK6HIrdx1JIm5AIl4lh1I+aLhJYm2yXIzXbLAGDV8F/3QgdIgRJXG1co3q9dAEmPjBeHRGxxkC6GKYIcNtZHJpYSTVtVLuCp0TMtr/WyXTA015bKK3Iu7FZ66jiwWrBT6muJle7RvwPsaEkP0eZZortap8bZUhnHZYuDM2BjZTcYJULcMuaS436sXMXqUJ/uPnOD5rH3m+r7IfMTVAzuIU9d2LKiLqOVH8kL8="
        ansible_user: rangeuser
        prefix: 24
        gateway: 10.0.5.2
        name_server: 10.0.5.5
        domain: blue1.local
        # Root causes the problem, it's either unreachable or the user can't sudo, even as an admin sudoer. Otherwise, it seems everything works fine.